common/prettify.pm

```diff
@@ -534,10 +557,17 @@ sub Warning ($)
 
         $self->Print_Sections ();
 
-        print {$self->{FH}} "<a name=\"warning_$counter\"></a>\n";
-        print {$self->{FH}} "<tt>[<a href=\"$self->{FULLHTML}#warning_$counter"
-                            . "\">Details</a>] </tt>";
-        print {$self->{FH}} "<font color=\"FF7700\"><tt>$s</tt></font><br>\n";
+        my $Warning1 = "<a name=\"warning_$counter\"></a>\n";
+        my $Warning2 = "<tt>[<a href=\"$self->{FULLHTML}#warning_$counter" . "\">Details</a>] </tt>";
```

Comment:
```suggestion
        my $Warning2 = "<tt>[<a href=\"$self->{FULLHTML}#warning_$counter\">Details</a>] </tt>";
```
As these are now on the same line you don't need to append them together and they can be one set of quotes.